sensor_msgs/PointCloud2 cloud
---
octomap_msgs/Octomap octomap
